sheet_to_triples/rdf.py

```diff
@@ -108,9 +108,13 @@ def graph_from_model(model):
     return g
 
 
+def _maybe_bnode(s, p, o):
+    return type(s) == rdflib.term.BNode or type(o) == rdflib.term.BNode
```

Comment:
Conventional spelling here is `isinstance(s, rdflib.term.BNode)` as the more OO-happy way.